2021-05-11 22:05:02,681 INFO    MainThread:7942 [wandb_setup.py:_flush():69] setting env: {}
2021-05-11 22:05:02,682 INFO    MainThread:7942 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-11 22:05:02,682 INFO    MainThread:7942 [wandb_init.py:_log_setup():336] Logging user logs to /home/pi/FedML-IoT/raspberry_pi/fedopt/wandb/run-20210511_220502-avddkqig/logs/debug.log
2021-05-11 22:05:02,682 INFO    MainThread:7942 [wandb_init.py:_log_setup():337] Logging internal logs to /home/pi/FedML-IoT/raspberry_pi/fedopt/wandb/run-20210511_220502-avddkqig/logs/debug-internal.log
2021-05-11 22:05:02,683 INFO    MainThread:7942 [wandb_init.py:init():369] calling init triggers
2021-05-11 22:05:02,683 INFO    MainThread:7942 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {'dataset': 'mnist', 'data_dir': '"./../../../data/mnist"', 'partition_method': 'hetero', 'partition_alpha': 0.5, 'model': 'lr', 'client_num_per_round': 2, 'comm_round': 200, 'epochs': 10, 'lr': 0.03, 'wd': 0.001, 'batch_size': 10, 'frequency_of_the_test': 1, 'is_mobile': 1}
2021-05-11 22:05:02,683 INFO    MainThread:7942 [wandb_init.py:init():418] starting backend
2021-05-11 22:05:02,683 INFO    MainThread:7942 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-11 22:05:02,708 INFO    MainThread:7942 [backend.py:ensure_launched():132] starting backend process...
2021-05-11 22:05:02,724 INFO    MainThread:7942 [backend.py:ensure_launched():137] started backend process with pid: 7961
2021-05-11 22:05:02,728 INFO    MainThread:7942 [wandb_init.py:init():423] backend started and connected
2021-05-11 22:05:02,731 INFO    MainThread:7942 [wandb_init.py:init():465] updated telemetry
2021-05-11 22:05:02,733 INFO    MainThread:7942 [wandb_init.py:init():484] communicating current version
2021-05-11 22:05:04,599 INFO    MainThread:7942 [wandb_init.py:init():489] got version response 
2021-05-11 22:05:04,599 INFO    MainThread:7942 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-11 22:05:04,806 INFO    MainThread:7942 [wandb_init.py:init():522] starting run threads in backend
2021-05-11 22:05:04,893 INFO    MainThread:7942 [wandb_run.py:_console_start():1565] atexit reg
2021-05-11 22:05:04,895 INFO    MainThread:7942 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-05-11 22:05:04,896 INFO    MainThread:7942 [wandb_run.py:_redirect():1417] Redirecting console.
2021-05-11 22:05:04,900 INFO    MainThread:7942 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-11 22:05:04,900 INFO    MainThread:7942 [wandb_init.py:init():546] run started, returning control to user process
2021-05-11 22:06:23,624 INFO    MainThread:7942 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 1
2021-05-11 22:06:23,625 INFO    MainThread:7942 [wandb_run.py:_restore():1507] restore
2021-05-11 22:06:25,662 INFO    MainThread:7942 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 770
  total_bytes: 770
}

2021-05-11 22:06:25,772 INFO    MainThread:7942 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 770
  total_bytes: 770
}

2021-05-11 22:06:25,946 INFO    MainThread:7942 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 770
  total_bytes: 770
}

2021-05-11 22:06:26,932 INFO    MainThread:7942 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 770
  total_bytes: 4357
}

2021-05-11 22:06:27,133 INFO    MainThread:7942 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 770
  total_bytes: 4357
}

2021-05-11 22:06:27,238 INFO    MainThread:7942 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4357
  total_bytes: 4357
}

2021-05-11 22:06:27,342 INFO    MainThread:7942 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4357
  total_bytes: 4357
}

2021-05-11 22:06:27,446 INFO    MainThread:7942 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4357
  total_bytes: 4357
}

2021-05-11 22:06:27,550 INFO    MainThread:7942 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4357
  total_bytes: 4357
}

2021-05-11 22:06:27,654 INFO    MainThread:7942 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4357
  total_bytes: 4357
}

2021-05-11 22:06:28,979 INFO    MainThread:7942 [wandb_run.py:_show_files():1879] logging synced files
